packages/core/interfaces/src/index.ts

```diff
@@ -207,11 +207,14 @@ export interface CheckoutGetters<SHIPPING_METHOD> {
   [getterName: string]: (element: any, options?: any) => unknown;
 }
 
-export interface UserOrderGetters<ORDER> {
+export interface UserOrderGetters<ORDER, ORDER_ITEM> {
   getDate: (order: ORDER) => string;
   getId: (order: ORDER) => string;
   getStatus: (order: ORDER) => string;
-  getPrice: (order: ORDER) => number;
+  getPrice: (order: ORDER) => AgnosticPrice;
+  getItems: (order: ORDER) => ORDER_ITEM[];
+  getBillingAddress: (address: ORDER) => string;
+  getShippingAddress: (address: ORDER) => string;
```

Comment:
would change that for implementation to decide about address structure
```suggestion
  getBillingAddress: (address: ORDER) => ADDRESS;
  getShippingAddress: (address: ORDER) => ADDRESS;
```

Reply:
remove and create RFC, ideally before 4 m so we can discuss it on decision making window @defudef